inkremental-viewpager2/src/main/kotlin/dev.inkremental.dsl.androidx.viewpager2/InkrementalViewPager2Setter.kt

```diff
@@ -0,0 +1,80 @@
+package dev.inkremental.dsl.androidx.viewpager2
+
+import android.view.View
+import androidx.recyclerview.widget.DiffUtil
+import androidx.viewpager2.widget.ViewPager2
+import dev.inkremental.Inkremental
+import dev.inkremental.attr
+import dev.inkremental.bind
+import dev.inkremental.dsl.androidx.recyclerview.HolderAttr
+import dev.inkremental.dsl.androidx.recyclerview.HolderAttrDiffable
+import dev.inkremental.dsl.androidx.recyclerview.InkrementalDiffCallback
+import dev.inkremental.dsl.androidx.recyclerview.RenderableRecyclerViewAdapter
+import dev.inkremental.dsl.androidx.viewpager2.widget.ViewPager2Scope
+import dev.inkremental.v
+
+fun viewPager2Inkremental(configure: InkrementalViewPager2Scope.() -> Unit = {}) =
```

Comment:
Maybe just `pager`? I like current naming on `recyclerView` / `list` we have for default / custom DSL for that, so I feel it could be applied here also. Less wordy, more natural. WDYT?

Reply:
👌